server/src/main/scala/za/co/absa/atum/server/api/database/flows/functions/GetFlowCheckpointsV2.scala

```diff
@@ -52,14 +56,27 @@ class GetFlowCheckpoints(implicit schema: DBSchema, dbEngine: DoobieEngine[Task]
     "measured_columns",
     "measurement_value",
     "checkpoint_start_time",
-    "checkpoint_end_time"
+    "checkpoint_end_time",
+    "has_more"
   )
 }
 
-object GetFlowCheckpoints {
-  val layer: URLayer[PostgresDatabaseProvider, GetFlowCheckpoints] = ZLayer {
+object GetFlowCheckpointsV2 {
+  case class GetFlowCheckpointsArgs(
+     flowId: Long,
+     limit: Option[Int],
+     offset: Option[Long],
+     checkpointName: Option[String]
+  )
+
+  object GetFlowCheckpointsArgs {
```

Comment:
Why did you define this serde stuff? It's not needed.

Reply:
Yes, they are not. Forgot to remove them.